src/light.cpp

```diff
@@ -24,38 +24,38 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 
 #ifndef SERVER
 
-// Length of LIGHT_MAX+1 means LIGHT_MAX is the last value.
+// Length of LIGHT_MAX + 1 means LIGHT_MAX is the last value.
 // LIGHT_SUN is read as LIGHT_MAX from here.
+u8 light_LUT[LIGHT_MAX + 1];
```

Comment:
Why not `static u8 light_LUT[LIGHT_SUN + 1];`?

Reply:
Static isn't used because it seems the ability to update the light table is intended (but not currently used)
"Initialize or update the light value tables using the specified gamma"
I'd rather use LIGHT_MAX here because that is used throughout this file.